<launch>
    <node pkg="multi_csrt" name="csrTracking" type="multi_csrt" output="screen">
        <param name="view_image"                value="true"/>
        <!-- track param -->
        <param name="min_hits"                  value="3"/>
        <param name="max_age"                   value="120"/>
        <!-- csrt param -->
        <param name="psr_thresh"                value="0.035"/>
        <param name="padding"                   value="3.0"/>
        <param name="filter_lr"                 value="0.02"/>
        <param name="template_size"             value="200"/>

        <param name="input_topic1"              value="/yolo/bbx"/>
        <param name="input_topic2"              value="/yolo/img"/>
        <param name="output_topic"              value="/result"/>
    </node>

    <!-- <node pkg="rviz" name="rviz" type="rviz" args="-d $(find projection_search)/proj.rviz" required="false"/> -->
</launch>
